client/src/Modules/Family/__test__/familyContainer.test.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { render, fireEvent, act } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import FamilyContainer from '../FamilyContainer.js';
+
+test('it should render without errors', () => {
+  expect(() => {
+    render(
+      <Router>
+        <FamilyContainer />
+      </Router>
+    );
+  }).not.toThrowError();
+});
+
+test(`should show 'Passwords should match!' if passwords do not match`, async () => {
```

Comment:
This is not implemented at `FamilyContainer` but rather is down stream at `PrimaryInfoFormComponent`. However testing this here follows more of what the user would do versus testing within the component.